### PR TITLE
[Term Entry] Java ArrayList method: .removeRange()

### PR DESCRIPTION
**Pull request by JA-FY**
### Description

- Added the ArrayList.removeRange() term for Java

### Issue Solved

Closes Issue #4344 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Comment by ishg-153** [comment]
Hi @JA-FY ! While I work on reviewing the entry please sign the CLA.

File: content/java/concepts/array-list/terms/removeRange/removeRange.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+The `.removeRange()` method is used for removing all elements inside a specified range from an instance of the ArrayList class.
+
+## Syntax
+
+`Elements` within a range can be removed from an `ArrayList` object by passing the range as int parameters to the `.removeRange()` method:
+
+```pseudo
+arrayListInstance.removeRange(int fromIndex, int toIndex);
+```
+
```
Comment (ishg-153) [inline content/java/concepts/array-list/terms/removeRange/removeRange.md:24]:
```suggestion

>**Note:**`fromIndex` and `toIndex` are both inclusive in the elements removed from the `ArrayList`.

```

**Review by ishg-153 — CHANGES_REQUESTED** [review]
@JA-FY I've suggested 2 changes, please make them. Let me know once done.

**Comment by JA-FY** [comment]
Changes made!

**Review by ishg-153 — APPROVED** [review]
@JA-FY This is good for a second review.

File: content/java/concepts/array-list/terms/removeRange/removeRange.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
```
Comment (avdhoottt) [inline content/java/concepts/array-list/terms/removeRange/removeRange.md:24]: Can you please describe the parameters in one line below the syntax block?

File: content/java/concepts/array-list/terms/removeRange/removeRange.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  - 'Computer Science'
+Tags:
+  - 'Arrays'
+  - 'Data Types'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The `.removeRange()` method is used for removing all elements inside a specified range from an instance of the ArrayList class.
```
Comment (avdhoottt) [inline content/java/concepts/array-list/terms/removeRange/removeRange.md:15]:
```suggestion
The `.removeRange()` method is used for removing all elements inside a specified range from an instance of the `ArrayList` class.
```
Link the codecademy's [ArrayList](https://www.codecademy.com/resources/docs/java/array-list) URL here.

File: content/java/concepts/array-list/terms/removeRange/removeRange.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+## Syntax
+
+`Elements` within a range can be removed from an `ArrayList` object by passing the range as int parameters to the `.removeRange()` method:
```
Comment (avdhoottt) [inline content/java/concepts/array-list/terms/removeRange/removeRange.md:19]:
```suggestion
Elements within a range can be removed from an `ArrayList` object by passing the range as `int` parameters to the `.removeRange()` method:
```

File: content/java/concepts/array-list/terms/removeRange/removeRange.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+```pseudo
+arrayListInstance.removeRange(int fromIndex, int toIndex);
+```
+
+**Note:**`fromIndex` and `toIndex` are both inclusive in the elements removed from the `ArrayList`.
```
Comment (avdhoottt) [inline content/java/concepts/array-list/terms/removeRange/removeRange.md:25]:
```suggestion
> **Note:** `fromIndex` and `toIndex` are both inclusive in the elements removed from the `ArrayList`.
```

File: content/java/concepts/array-list/terms/removeRange/removeRange.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+    tasksList.removeRange(0, 3);
+
+    // Print the list again to see the differences
+    System.out.println("List with elements removed: " + tasksList);
+   
+  }
+}
+```
+
+The output would look like this:
```
Comment (avdhoottt) [inline content/java/concepts/array-list/terms/removeRange/removeRange.md:59]:
```suggestion
The output will look like this:
```

**Review by avdhoottt — CHANGES_REQUESTED** [review]
Hey @JA-FY, I have suggested some changes, please make them.

**Comment by ishg-153** [comment]
@JA-FY It's been two weeks since @avdhoottt suggested changes. Are you still working on them?

File: content/java/concepts/array-list/terms/removeRange/removeRange.md (hunk 3 above)
Comment (ishg-153) [reply content/java/concepts/array-list/terms/removeRange/removeRange.md]: Didn't add the link as this entry is under the same concept.

**Comment by don-mo** [comment]
@ishg-153 and @avdhoottt what does the following mean?
image

File: content/java/concepts/array-list/terms/removeRange/removeRange.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+  - 'Computer Science'
+Tags:
+  - 'Arrays'
+  - 'Data Types'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The `.removeRange()` method removes all elements inside a specified range from an instance of the `ArrayList` class.
```
Comment (avdhoottt) [inline content/java/concepts/array-list/terms/removeRange/removeRange.md:15]:
```suggestion
The **`.removeRange()`** method removes all elements inside a specified range from an instance of the `ArrayList` class.
```

**Review by avdhoottt — APPROVED** [review]
LGTM

**Pull request merged**